helm/alloy/examples/mimir-rules/values.yaml

```diff
@@ -0,0 +1,17 @@
+alloy:
+  alloy:
```

Comment:
I think it would bé cool to add an example with évents as well

Reply:
We can add this later on, examples with host files would be nice too, like audit logs.